TestFoundation/TestNSOrderedSet.swift

```diff
@@ -135,4 +149,185 @@ class TestNSOrderedSet : XCTestCase {
         XCTAssertEqual(set.count, 2)
         XCTAssertEqual(set.indexOfObject("baz" as NSString), 1)
     }
-}
\ No newline at end of file
+
+    func test_IsEqualToOrderedSet() {
+        let set = NSOrderedSet(array: ["foo", "bar", "baz"].bridge().bridge())
+        let otherSet = NSOrderedSet(array: ["foo", "bar", "baz"].bridge().bridge())
+        let otherOtherSet = NSOrderedSet(array: ["foo", "bar", "123"].bridge().bridge())
+        XCTAssert(set.isEqualToOrderedSet(otherSet))
+        XCTAssertFalse(set.isEqualToOrderedSet(otherOtherSet))
+    }
+
+    func test_Subsets() {
+        let set = NSOrderedSet(array: ["foo", "bar", "baz"].bridge().bridge())
+        let otherOrderedSet = NSOrderedSet(array: ["foo", "bar"].bridge().bridge())
+        let otherSet = Set(["foo".bridge(), "baz".bridge()])
+        let otherOtherSet = Set(["foo".bridge(), "bar".bridge(), "baz".bridge(), "123".bridge()])
+        XCTAssert(otherOrderedSet.isSubsetOfOrderedSet(set))
+        XCTAssertFalse(set.isSubsetOfOrderedSet(otherOrderedSet))
+        XCTAssertFalse(set.isSubsetOfSet(otherSet))
```

Comment:
this line also crashes the compiler:
```
1.	While emitting SIL for 'test_Subsets' at TestFoundation/TestNSOrderedSet.swift:161:5
2.	While silgen closureexpr SIL function @_TFFC14TestFoundation16TestNSOrderedSet12test_SubsetsFT_T_u1_KT_Ps11BooleanType_ for expression at [TestFoundation/TestNSOrderedSet.swift:168:24 - line:168:50] RangeText="set.isSubsetOfSet(otherSet)"
```

Not certain what to do as a work-around here